src/setup_runs/config_read_functions.py

```diff
@@ -0,0 +1,311 @@
+import os
+import sys
+import json
+import re
+from typing import Tuple
+import datetime
+import pytz
+from attrs import define
+
+
+@define
+class WRFConfig :
+    project_root: str  # overall base directory for project
```

Comment:
```suggestion
    project_root: str
    """overall base directory for project"""
```

Then the docs work better